app/controllers/admin/projects_controller.rb

```diff
@@ -18,7 +18,10 @@ class Admin::ProjectsController < Channels::Admin::BaseController
 
   def index
     respond_to do |format|
-      format.html { collection }
+      format.html do
+        @movable_projects_to_channel = Project.movable_to_channel.order(:name)
+        collection
```

Comment:
I know that it is a legacy code, but I really dislike this kind of approach. I mean, call a method name `collection` suddenly and alone like this is awkward IMO. Would it break the followed pattern  on the rest of the class if you instantiate the IV directly here?